Improve Kotlin tour (#4321)
JetBrains/kotlin-web-site: docs/topics/tour/kotlin-tour-classes.md

|---|---|
```kotlin
data class Person(val name: Name, val address: Address, val ownsAPet: Boolean = true)
// Write your code here
// data class Name(...)

fun main() {
    val person = Person(
        Name("John", "Smith"),
        Address("123 Fake Street", City("Springfield", "US")),
        ownsAPet = false
    )
}
```
{validate="false" kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-tour-classes-exercise-2"}

|---|---|
```kotlin
data class Person(val name: Name, val address: Address, val ownsAPet: Boolean = true)
data class Name(val first: String, val last: String)
data class Address(val street: String, val city: City)
data class City(val name: String, val countryCode: String)

fun main() {
    val person = Person(
        Name("John", "Smith"),
        Address("123 Fake Street", City("Springfield", "US")),
        ownsAPet = false
    )
}
```
{initial-collapse-state="collapsed" collapsed-title="Example solution" id="kotlin-tour-classes-solution-2"}

### Exercise 3 {initial-collapse-state="collapsed"}

To test your code, you need a generator that can create random employees. Define a `RandomEmployeeGenerator` class with 

> a fixed list of potential names (inside the class body), and that is configured by a minimum and maximum salary (inside the class header). 

Reviewer: ```suggestion
a fixed list of potential names (inside the class body). Configure it by a minimum and maximum salary (inside the class header). 
```
